.travis.yml

```diff
@@ -5,27 +5,15 @@ language: go
 go: "1.13.x"
 go_import_path: github.com/google/trillian
 
-addons:
-  apt:
-    packages:
-      - softhsm
-
 cache:
   directories:
     - "$HOME/.cache/go-build"
     - "$HOME/.cache/go-build-race"
     - "$HOME/google-cloud-sdk/"
     - "$HOME/gopath/pkg/mod"
 
-env: #TODO: Merge the test matrix into individual jobs
-  - DO_NOTHING_SPECIAL=true # The first line of env will be used for everything in jobs, below.
-  - PRESUB_TESTS=true GOFLAGS='-race'                      PRESUBMIT_OPTS="--no-linters --no-generate"
-  - PRESUB_TESTS=true GOFLAGS='-race --tags=batched_queue' PRESUBMIT_OPTS="--no-linters --no-generate"
-  - PRESUB_TESTS=true GOFLAGS='-race --tags=pkcs11' WITH_PKCS11=true PRESUBMIT_OPTS="--no-linters --no-generate"
-  - INTEG_TESTS=true GOFLAGS='-race'                       PRESUBMIT_OPTS="--no-linters --no-generate"
-  - INTEG_TESTS=true GOFLAGS='-race --tags=batched_queue'  PRESUBMIT_OPTS="--no-linters --no-generate"
-  - INTEG_TESTS=true GOFLAGS='-race' WITH_ETCD=true        PRESUBMIT_OPTS="--no-linters --no-generate"
-  - INTEG_TESTS=true GOFLAGS='-race --tags=pkcs11' WITH_PKCS11=true PRESUBMIT_OPTS="--no-linters --no-generate"
+env:
+  GO_TEST_TIMEOUT=20m
```

Comment:
Did you mean this to be a test line or a global environment setting?

Reply:
This used to be exported in the common script: section. I don't see a problem with having it in the global env, but we can put it explicitly in the presubmit and integration sections if you prefer? These scripts both read this env variable.

Reply:
Should this look like
```
env:
    global:
       GO_TEST_TIMEOUT=20m
```

I'm just worried that travis will interpret this as another dimension rather than as just setting a variable.